cfApplication.go

```diff
@@ -48,6 +51,65 @@ const (
 	TCP   RouteProtocol = "tcp"
 )
 
-type ServiceSpec struct {
+// Services represents a slice of Service
+type Services []Service
+
+// Service contains the specification for an existing Cloud Foundry service required by the application
+type Service struct {
+	// Name represents the name of the Cloud Foundry service required by the application. This field
+	// represents the runtime name of the service, captured from the 3 different cases where
+	// the service name can be listed.
+	// For more information check https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#services-block
+	Name string `json:"name"`
+	// Parameters contain the k/v relationship for the aplication to bind to the service
+	Parameters map[string]interface{} `json:"parameters,omitempty"`
+	// Tag
+}
+
+// Processes represents a slice of Processes.
+type Processes []Process
+
+// Process represents the abstraction of the specification of a Cloud Foundry Process.
+// For more information check https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#processes
+type Process struct {
+	// Name represents the name of the process.
+	Name string `json:"name"`
+	// Image represents the pull spec of the container image.
+	Image string `json:"image"`
+	// Memory represents the amount of memory requested by the process.
+	Memory string `json:"memory,omitempty"`
+	// DiskQuota represents the amount of persistent disk requested by the process.
+	DiskQuota string `json:"disk,omitempty"`
+	// HealthCheck captures the health check information
+	HealthCheck Probe `json:"healthCheck"`
+	// ReadinessCheck captures the readiness check information.
+	ReadinessCheck ReadinessProbe `json:"readinessCheck"`
+	// Command represents the command used to run the process.
+	Command []string `json:"command,omitempty"`
```

Comment:
doubt: in CF there are ProcFiles. Is this field enough to capture them?

Reply:
What do you mean `ProcFiles`? process files?

Reply:
CF uses [ProcFiles](https://v3-apidocs.cloudfoundry.org/version/3.184.0/#procfiles) to declare required runtime processes. As far as i understood, they are used for buildpacks (and in our context doesn't apply) or to define the `web` process (that reading the examples is used) or to define custom runtime processes. So the doubt regards the possible needs to map this aspect.

Reply:
I've added `Type` to the `Process` structure as discussed. PTAL